frontend/packages/core/src/NPS/emojiRatings.tsx

```diff
@@ -27,11 +32,18 @@ const getKey = (map, val): string => Object.keys(map).find(key => map[key] === v
  * @param setRating function which will return the given selected rating
  * @returns rendered EmojiRatings component
  */
-const EmojiRatings = ({ ratings = [], setRating }: EmojiRatingsProps) => {
+const EmojiRatings = ({
+  ratings = [],
+  setRating,
+  placement = "top",
+  buttonSize = "small",
+  emojiSize = "medium",
+}: EmojiRatingsProps) => {
   const [selectedRating, selectRating] = useState<Rating>(null);
 
   const StyledIconButton = styled(IconButton)<{
     $selected: boolean;
+    size: string;
```

Comment:
Just confirming, do you want `size` to be forwarded to the `IconButton` component?

Reply:
Yep, the IconButton needs the size but then the IconButton will handle filtering out the transient prop any further.